### DISCUSSION TITLE
Multi_Labels_Classification_models

### DISCUSSION THREAD

**Discussion by kikiru328** [Pleasant-riot/Lung-Disease-Detection#22] [선행연구]
0. TJnet  변형모델 ( block 5 --> AGE)
![image](https://user-images.githubusercontent.com/60537388/143386440-0c6750c5-b01e-4604-a001-3c2b88ddb0bf.png)

1 . LibAUC model
https://github.com/Optimization-AI/LibAUC
https://arxiv.org/pdf/2012.03173v2.pdf
- 외부 모듈로 만들어진 딥러닝 모델.
- 아키텍쳐 및 내부 구성 공부 필수 ( 없으면 사용 불가)
- 의료용 데이터를 활용할 수 있도록 함

2. query2labels  -->  Q2L-CvT
https://github.com/SlongLiu/query2labels
- transfomer 모델 분류 ( vit )
- COCO데이터 사용, classification ROC_AUC 0.91 
- transformer 모델 공부

3. CNN model for chest x-ray
https://arxiv.org/pdf/1911.06475v3.pdf

**Comment by SHEYCHAE** [comment]
tjent 누구네 피피티인지 참~ 잘 만들었네요
